gtars/src/uniwig/mod.rs

```diff
@@ -240,9 +264,23 @@ pub fn uniwig_main(
             }
         };
 
+        final_chromosomes.push(chromosome.clone())
+    }
+
+    println!("Initial chroms: {}  vs Final chroms: {}", chromosomes.len(), final_chromosomes.len());
+
+    final_chromosomes.par_iter().with_min_len(8).for_each(|chromosome: &Chromosome|
+        {
+
+        // Need these for setting wiggle header
+        let primary_start = chromosome.starts[0].clone();
+        let primary_end = chromosome.ends[0].clone();
```

Comment:
Here you are calling `clone()` on an `i32` which actually implements `Copy` so its unnecessary

Reply:
`i32`'s are of a known size at compile time so Rust pushes and pops off the stack for these. So `clone` is redundant